src/internal/pachd/testpachd.go

```diff
@@ -184,6 +186,9 @@ func NewTestPachd(t testing.TB, opts ...TestPachdOption) *client.APIClient {
 	}()
 	pd.AwaitAuth(ctx)
 	for _, opt := range opts {
+		if opt.CopyDBConfig != nil {
+			*opt.CopyDBConfig = dbcfg
+		}
```

Comment:
Just so you know, there is pachsql.ConnStringFromConn that returns the DSN from a pachsql.DB object.  This is targeted at connecting with psql and pg_dump but probably works for reconnecting here.

Reply:
I took a quick shot at that, but it hangs with an error while starting up. I started digging, but decided it can wait for another day.